server/src/index.js

```diff
@@ -26,8 +26,10 @@ const users = cache.create(async (user) => {
   return {...u.profile, id: user}
 })
 const channels = cache.create(async (channel) => {
-  const c = await web.conversations.info({channel})
-  return c.channel
+  const c = await web.conversations.info({channel}).catch((error) => {
+    logger.error(`Error while loading channel ${channel}: `, error)
```

Comment:
This will quite heavily spam the logs. Shouldn't we clean-up the db? Or should the messages be kept in report-bot?

Reply:
Let's keep the reports and skip logging if the error was 'channel_not_found'.